spec/index.md

```diff
@@ -193,50 +198,18 @@ This follows [Data Package][dp] (DP). In particular, the following properties `M
 ```
 
 
-
-## Resources
```

Comment:
Looking at this again, is there a reason you wanted to remove this?  Most people aren't necessarily aware of the requirements for data packages generally.  I'm actually wondering if it wouldn't be more helpful to keep this text, and then include a very short example of what the "resources" key could map to.  This example resource could then be referred to throughout the spec as we build out what minimal measures and dimensions objects would look like.

If you add this paragraph back, I can merge, and then I can work on this example resources object.